framework/SupervisedLearning/ARMA.py

```diff
@@ -1139,8 +1141,16 @@ def __returnCurrentSettingLocal__(self):
     pass
 
   def setEngine(self,eng,seed=None,count=None):
+    """
+     Set up the random engine for arma
+     @ In, engine, instance, random number generator
+     @ In, seed, int, optional, the seed, if None then use the global seed from ARMA
+     @ In, count, int, optional, advances the state of the generator, if None then use the current ARMA.randomEng count
+     @ Out, engine, instance, random number generator
```

Comment:
It looks like the input argument is `eng` instead of `engine`, and nothing is returned, right? 

This method could be modified to return the engine; since it gets seeded and forwarded, that could be useful potentially.